source/switching_to_live/set_up_a_live_stripe_account/index.html.md.erb

```diff
@@ -0,0 +1,25 @@
+---
+title: Connect your live account to Stripe
+weight: 100
+---
+
+# Connect your live account to Stripe
+
+First make sure you've:
+
+- read about [what to do first](/switching_to_live/#before-you-start)
+- [requested a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
```

Comment:
maybe we add a link to the how to use Stripe docs as well? https://www.payments.service.gov.uk/payment-service-provider/

Reply:
I think we expect that developers won't need the info in the product pages, as by the time they're integrating/going live, the product pages will already have been digested by other members of the service team. So it's probably best to leave this as is - however I'm very keen to see developer user journeys in real life to test this assumption!

Reply:
It's possible that a small service without a technical team might end up in the docs if they are looking at how to go live with Stripe? 

And it might be that the technical team are building some of the automatic reconciliation processes for the finance team, and would need to know things like when the payments will be deposited into their account. I can forward an email chain from one service where the developer had a lot of questions on that. (sorry for not thinking about it before!)

Reply:
Hi Miriam - changes as discussed in https://github.com/alphagov/pay-tech-docs/pull/356/commits/e9bf6d29ab93cfba4848afccd8605cefe780eeff (added links to fees and receiving payments within the tech docs).

We also discussed updates on the Payment flow page around payout times and bank statement text. I'll create separate stories for those.

Reply:
Looks good! Will resolve the conversation